### PR TITLE
Make shift clicks on the currentPath element copy the relative path

### PR DESCRIPTION
**Pull request by awans**
Implement ctrl/cmd click -> copy relative path as discussed in [this issue](https://github.com/atom/status-bar/issues/104).

I've added the most obvious test, but let me know if there are others you'd like me to take a pass at..

Please also let me know if I've missed any subtle issues!

This is my first:
- atom contribution
- coffeescript
- atom spec

so I'd love to learn :)

### CODE REVIEW

File: lib/file-info-view.coffee [hunk 1]
```diff
@@ -9,35 +9,43 @@ class FileInfoView extends HTMLElement
...
-    @handleCopiedTooltip()
-
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
-    clickHandler = =>
-      text = @getActiveItemCopyText()
+    clickHandler = (event) =>
+      altClick = if process.platform is 'darwin' then event.metaKey else event.ctrlKey
```
Comment (Daniel15) [inline lib/file-info-view.coffee]: Why does this need to differ by platform? Can't you just make it `Ctrl` for all platforms?

Comment (awans) [inline lib/file-info-view.coffee]: We could, but I'd rather not from a UX perspective -- control clicking on OS X is a lot less natural than Command clicking; this is the pattern that Facebook's `hyperclick` package uses, too!

Comment (50Wliu) [inline lib/file-info-view.coffee]: I think it's fine the way it is right now - `cmd` is equivalent to `ctrl` on Windows/Linux.

**Comment by thedaniel** [comment]
@awans This looks pretty good - I think `altClick` is kind of a confusing variable name since `alt` is another modifier key, and it's not clear that it's a boolean. Anyway, I left a comment over on #104 with some concerns about the feature in general, but if those are resolved positively this probably only needs that minor change to merge.

File: lib/file-info-view.coffee (hunk 1 above)
Comment (thedaniel) [inline lib/file-info-view.coffee]: cf https://github.com/atom/status-bar/issues/104#issuecomment-180125702 for my feelings on it

File: lib/file-info-view.coffee [hunk 2]
```diff
@@ -9,35 +9,43 @@ class FileInfoView extends HTMLElement
...
-    text = @getActiveItemCopyText()
+    text = @getActiveItemCopyText(isShiftClick)
     @copiedTooltip = atom.tooltips.add this,
       title: "Copied: #{text}"
       trigger: 'click'
       delay:
         show: 0
 
-  getActiveItemCopyText: ->
+  getActiveItemCopyText: (isShiftClick) ->
```
Comment (benogle) [inline lib/file-info-view.coffee]: Can you rename this from `isShiftClick` to `useRelativePath` for clarity?

Comment (benogle) [inline lib/file-info-view.coffee]: I'll change the name on my end

**Comment by benogle** [comment]
Thanks!

File: lib/file-info-view.coffee (hunk 2 above)
Comment (awans) [inline lib/file-info-view.coffee]: sgtm, and thanks!

**Comment by danielbayley** [comment]
Feels like this should be _alt_ <kbd>⌥</kbd> click surely?

**Pull request merged**